src/libraries/System.Linq/src/System/Linq/SelectManyWithParameterHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Linq
+{
+    internal static class SelectManyWithParameterHelpers
```

Comment:
This file should not be placed under `System.Linq` since it's not part of that assembly.